src/main/java/com/mraof/minestuck/item/weapon/aspect/BreathAOEWeaponItem.java

```diff
@@ -0,0 +1,59 @@
+package com.mraof.minestuck.item.weapon.aspect;
+
+import com.mraof.minestuck.item.weapon.KnockbackWeaponItem;
+import com.mraof.minestuck.item.weapon.MSToolType;
+import com.mraof.minestuck.player.EnumAspect;
+import com.mraof.minestuck.player.Title;
+import com.mraof.minestuck.world.storage.PlayerSavedData;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.item.IItemTier;
+import net.minecraft.item.ItemStack;
+import net.minecraft.particles.ParticleTypes;
+import net.minecraft.potion.EffectInstance;
+import net.minecraft.potion.Effects;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.SoundEvents;
+import net.minecraft.util.math.AxisAlignedBB;
+
+import java.util.List;
+
+public class BreathAOEWeaponItem extends KnockbackWeaponItem
+{
+	private final EnumAspect aspect;
+	
+	public BreathAOEWeaponItem(IItemTier tier, int attackDamageIn, float attackSpeedIn, float efficiency, EnumAspect aspect, MSToolType toolType, Properties builder)
+	{
+		super(tier, attackDamageIn, attackSpeedIn, efficiency, toolType, builder);
+		this.aspect = aspect;
+	}
+	
+	@Override
+	public boolean hitEntity(ItemStack stack, LivingEntity target, LivingEntity attacker)
+	{
+		if(attacker instanceof ServerPlayerEntity)
+		{
+			Title title = PlayerSavedData.getData((ServerPlayerEntity) attacker).getTitle();
+			
+			if(title != null && attacker.getRNG().nextFloat() < .1)
+			{
+				if(title.getHeroAspect() == aspect){
+					AxisAlignedBB axisalignedbb = attacker.getBoundingBox().grow(4.0D, 2.0D, 4.0D);
+					List<LivingEntity> list = attacker.world.getEntitiesWithinAABB(LivingEntity.class, axisalignedbb);
+					if (!list.isEmpty()) {
+						for(LivingEntity livingentity : list) {
+							double d0 = attacker.getDistanceSq(livingentity);
+							if (d0 < 16.0D) {
+								attacker.world.playSound(null, attacker.getPosX(), attacker.getPosY(), attacker.getPosZ(), SoundEvents.ENTITY_ENDER_DRAGON_FLAP, SoundCategory.PLAYERS, 1.5F, 1.4F);
+								livingentity.addPotionEffect(new EffectInstance(Effects.LEVITATION, 60, 1));
+								attacker.world.addParticle(ParticleTypes.CLOUD, (float)attacker.getPosX() + random.nextFloat(), (float)attacker.getPosY() + random.nextFloat(), (float)attacker.getPosZ() + random.nextFloat(), target.getMotion().x, target.getMotion().y, target.getMotion().z);
+								attacker.removePotionEffect(Effects.LEVITATION);
```

Comment:
Looks rather similar to the effect for the CRUEL_FATE_CRUCIBLE weapon. Perhaps it is appropriate to make an utility function that takes care of this kind of thing? Also, the potion effect removal comment also applies here (perhaps even more so since it's more likely that a levitation effect was already present before the weapon did its thing.

Reply:
Im not sure I know how to make an acceptable utility function! But I would be willing to learn or accept changes you can reccomend

Reply:
Hmm, we can probably hold off on this for the weapon properties stuff